tensorflow/core/util/mkl_util.h

```diff
@@ -1853,13 +1893,59 @@ class MklPrimitiveFactory {
   ~MklPrimitiveFactory() {}
 
   MklPrimitive* GetOp(const string& key) {
+#ifndef DNNL_AARCH64_USE_ACL
     auto& lru_cache = MklPrimitiveFactory<T>::GetLRUCache();
     return lru_cache.GetOp(key);
+#else
+    while(true) {
```

Comment:
TODO comment added in ecb7614. 

We have wider scope because after threads (in our investigations we noticed there it can be more then one) that are waiting for primitive to be created need to go `GetOp()` the primitive again.